### PR TITLE
RE: Update install doc regarding elasticsearch timed out error

### LINKED ISSUE #14356: Update install doc regarding elasticsearch timed out error
Upon initial install, there is a ConnectTimeoutError - `Connection to elasticsearch timed out` and running `docker-compose ps` shows the elasticsearch container exits with code 137.

Screen Shot 2020-05-20 at 8 20 30 AM

Screen Shot 2020-05-20 at 8 26 36 AM

The install doc notes that increasing default RAM limit settings for those who installed with docker-for-windows only; however, this also applies to docker-for-mac as well.

In addition, updating the default settings should be noted when choosing to install with Docker.

### PR DESCRIPTION
**Pull request by xlisachan**
Fixes #14356 

The following revisions to the install doc have been made:

1. Added a line to prompt update of default RAM limit settings when selecting Docker to install project
2. Updated elasticsearch gotcha note from "Docker for Windows only: elasticsearch container exits with code 137" to the actual error message of "Connection to elasticsearch timed out"

### CODE REVIEW

File: docs/topics/install/docker.rst [hunk 1]
```diff
@@ -14,6 +14,7 @@ There are two options for running docker depending on the platform
 you are running.
 
  * Run docker on the host machine directly (recommended)
+    - Update default settings in Docker Desktop - increase RAM limit from 2 GB (preferably 4 GB) in the Resources/Advanced section and click on "Apply and Restart".
```
Comment (bobsilverberg) [inline docs/topics/install/docker.rst:17]: I find this sentence a bit hard to parse. Perhaps it should be "increase RAM limit from 2 GB (we've found that 4 GB works)...".

Comment (xlisachan) [reply docs/topics/install/docker.rst:17]: Can I update it to simply "... increase RAM limit from 2 GB to 4 GB..."?

**Comment by xlisachan** [comment]
@bobsilverberg Per our discussion, the wording has been updated. Please review when you have a moment. Thanks!
